mxcubeweb/core/components/lims.py

```diff
@@ -298,24 +297,19 @@ def get_proposal(self, user):
 
     def select_proposal(self, proposal):
         proposal_info = self.get_proposal_info(proposal)
-
-        if (
-            HWR.beamline.lims.loginType.lower() == "user"
-            and "Commissioning" in proposal_info["Proposal"]["title"]
-        ):
-            if hasattr(HWR.beamline.session, "set_in_commissioning"):
+        if proposal_info:
```

Comment:
the issue was TypeError (if proposal_info was None) or KeyError raised here:
```
and Commissioning" in proposal_info["Proposal"]["title"]
```
instead of returning False at the end, and finally it was carried and re-raised here: https://github.com/mxcube/mxcubeweb/blob/develop/mxcubeweb/routes/login.py#L43

Reply:
> ...user already appeared as active (https://github.com/mxcube/mxcubeweb/blob/develop/mxcubeweb/core/components/user/usermanager.py#L343)

I see, the first comment above threw me off a bit. So I imagine that this is with user based login ? Out of curiosity how come that `proposal_info` was None ?

Reply:
sorry for my mistake: empty dict not None